src/Resources/config/config.xml

```diff
@@ -117,4 +117,24 @@
             </helpText>
         </input-field>
     </card>
+    <card>
+        <title>Delayed Capture</title>
+        <input-field type="bool">
+            <name>active</name>
+            <label>Active</label>
+        </input-field>
+        <input-field type="int">
+            <name>rescheduleTime</name>
+            <label>Reschedule Notification Time</label>
+            <helpText>
+                This is the time to wait before trying the next capture in seconds
+            </helpText>
+            <defaultValue>43200</defaultValue>
+        </input-field>
+        <component name="sw-entity-single-select">
+            <name>orderState</name>
+            <entity>state_machine_state</entity>
```

Comment:
this entity lists all the states in the state machine (payment, delivery and order states) instead of only the delivery states. So the common options like `open`, `in_progress` and `cancelled` will appear 3 times in the dropdown, all with different state IDs

Reply:
this has been an issue in shopware a longer time apparently, added a quick solution as mentioned [here](https://forum.shopware.com/t/criteria-in-sw-entity-single-select-component-in-der-pluginkonfiguration-setzen/64873/10)